src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -357,19 +359,36 @@ func (cfg Config) doFetchCloudConfig(proxyAddr string) ([]byte, error) {
 	if err != nil {
 		return nil, fmt.Errorf("Unable to initialize HTTP client: %s", err)
 	}
+	country := strings.ToLower(globals.GetCountry())
```

Comment:
Right, but even if GetCountry returns "", we're out of luck right? It'll pick it up on the next pass, but that doesn't seem ideal @aranhoide.